bolt/org_config.go

```diff
@@ -0,0 +1,169 @@
+package bolt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/bolt/internal"
+)
+
+// Ensure OrganizationConfigStore implements chronograf.OrganizationConfigStore.
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+// OrganizationConfigBucket is used to store chronograf organization configurations
+var OrganizationConfigBucket = []byte("OrganizationConfigV1")
+
+// OrganizationConfigStore uses bolt to store and retrieve organization configurations
+type OrganizationConfigStore struct {
+	client *Client
+}
+
+func (s *OrganizationConfigStore) Migrate(ctx context.Context) error {
+	return nil
+}
+
+// FindOrCreate gets an OrganizationConfig from the store or creates one if none exists for this organization
+func (s *OrganizationConfigStore) FindOrCreate(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
+	var cfg chronograf.OrganizationConfig
+	err := s.client.db.View(func(tx *bolt.Tx) error {
+		v := tx.Bucket(OrganizationConfigBucket).Get([]byte(orgID))
+		if v == nil {
+			cfg = newOrganizationConfig(orgID)
+			return nil
+		}
+		return internal.UnmarshalOrganizationConfig(v, &cfg)
+	})
+
+	if err != nil {
+		return nil, err
+	}
+	return &cfg, nil
+}
+
+// Update replaces the OrganizationConfig in the store
+func (s *OrganizationConfigStore) Update(ctx context.Context, cfg *chronograf.OrganizationConfig) error {
```

Comment:
This update feels very `Put`-y. Is this what we do elsewhere?

Reply:
Yes, the `Put` route calls this method